rocrand/PKGBUILD

```diff
@@ -34,4 +34,7 @@ package() {
 /opt/rocm/rocrand/lib
 EOF
   install -Dm644 "$srcdir/rocRAND-rocm-$pkgver/LICENSE.txt" "$pkgdir/usr/share/licenses/$pkgname/LICENSE"
+  # Some packages search for hiprand includes in /opt/rocm/include/hiprand
+  install -d "$pkgdir/opt/rocm/include/hiprand"
```

Comment:
I believe this should be `install -d "$pkgdir/opt/rocm/include"`

Reply:
Actually no, it should be `ln -s /opt/rocm/hiprand/include/* "$pkgdir/opt/rocm/include/"` (because the target is `#include "rocm/hiprand/include/hiprand.h"`), but I'm guessing that removing the quotes on the first argument is a bad practice and the asterisk cannot be inside the quotes.

Reply:
We should symlink the entire folder. As of now none of the files are symlinked.
You can check by inspecting the `pkg/` contents of the built package after you `makepkg` it.

Reply:
Sure, but no `install` since while `hip` has `/opt/rocm/hip/include/hip/[file].h`, hiprand does not have the last folder: `/opt/rocm/hiprand/include/[file].h´.

Reply:
Absolutely, sorry. Meanwhile, I discovered I can't build rocrand, so I made an issue about it since I do not know how to fix oit

Reply:
I built and tested with `arch4edu-x86_64-build` and got:

```
$: /var/lib/archbuild/arch4edu-x86_64/fernando/build/rocrand/pkg/rocrand/opt/rocm/include/hiprand
   ls
hiprand.h      hiprand_kernel.h       hiprand_mtgp32_host.h
hiprand.hpp    hiprand_kernel_hcc.h   hiprand_nvcc.h
hiprand_hcc.h  hiprand_kernel_nvcc.h  hiprand_version.h
```

So I guess it is correct now

Reply:
@fernandoblalves it is still not working for me. After building the package I cannot `ls` into the symlinked `hiprand`.
`ln -s "/opt/rocm/rocrand/include/hiprand" "$pkgdir/opt/rocm/include/hiprand"` is the solution

Reply:
I am still not getting that but I believe the `install` command is now correct.